src/explore-education-statistics-admin/src/services/releasePermissionService.ts

```diff
@@ -0,0 +1,40 @@
+import client from '@admin/services/utils/service';
+
+export interface ManageAccessPageContributor {
+  userId: string;
+  userFullName: string;
+  userEmail: string;
+  releaseRoleId: string | undefined;
```

Comment:
As mentioned in the backend code, doesn't really make sense for us to have a `releaseRoleId`